src/pages/api/simulation.ts

```diff
@@ -0,0 +1,28 @@
+import { TENDERLY_ACCESS_KEY, TENDERLY_PROJECT, TENDERLY_USER } from "../../consts/config";
+
+
+export default async function handler(req,res){
+  const data=req.body
```

Comment:
Define a type interface for the expected body schema and use it here and in TransactionCard.tsx when forming the request

Reply:
here the body is a string so no need of using the type interface but used it in TransactionCard.tsx